starter-code/routes/celebrities.js

```diff
@@ -0,0 +1,71 @@
+const express = require("express");
+const Celebrity = require("../models/celebrity-model.js");
+const router = express.Router();
+
+router.get("/celebrities", (req, res, next) => {
+  Celebrity.find()
+    .sort({ rating: -1 })
```

Comment:
you didn't need to sort by rating because your celebrities don't have ratings

Reply:
Ah wrong copy paste from the lesson ><